index.js

```diff
@@ -237,6 +237,32 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		const killResult = originalKill(signal);
+		if (signal === undefined ||
```

Comment:
Could you please add a test for when `signal` is `undefined`? I.e. when using `kill()` with no arguments.

This would give you 100% test coverage on this PR: https://coveralls.io/builds/23196199/source?filename=index.js#L281

Reply:
Of course, added: c0c3527d06777617a30b6e4391442f5a5eef3244